packages/cli/README.md

```diff
@@ -1,29 +1,74 @@
 # Samen
 
-Samen is a toolkit for TypeScript backend development. It has a minimal API, it's fast and it generates a type-safe client for your frontend. There's no need to learn new languages or DSL's: Samen understands the code you already have, bringing your backend and frontend closer together than ever before:
+Samen is a new kind of backend framework for building RPC based API's in TypeScript. Our mission is to reduce the amount of (boilerplate) code you need to create a backend for your apps.
```

Comment:
When did we decide to go back to calling it a `framework` again. What happened to `toolkit`? 😝

Reply:
Haha good question. I'm not entirely sure on this one. But I'm guessing "framework" sounds a bit more familiar. Toolkit was inspired by React because it's more than a library, it comes with a TUI, a dev server, etc.

Reply:
As discussed offline we go with "framework" for now.